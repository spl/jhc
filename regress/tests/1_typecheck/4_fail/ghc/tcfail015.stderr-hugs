ERROR "tcfail015.hs":7 - Instance of Num Bool required for definition of g
