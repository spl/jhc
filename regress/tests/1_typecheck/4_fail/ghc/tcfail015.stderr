
tcfail015.hs:7:13:
    No instance for (Num Bool)
      arising from the literal `2'
    Possible fix: add an instance declaration for (Num Bool)
    In the expression: 2
    In an equation for `g': g (ANull) = 2
